spec/classes/windows_firewall/windows_firewall_spec.rb

```diff
@@ -165,15 +165,15 @@
     end
   end
 
-  context 'passing invalid param ensure: fubar' do
+  context 'passing invalid type for param ensure' do
     let :params do
-      { ensure: 'fubar' }
+      { ensure: ['not','a','string'] }
     end
 
     it do
       expect do
         is_expected.to contain_registry_value('EnableFirewall')
-      end.to raise_error(Puppet::Error)
+      end.to raise_error(Puppet::Error,/expects a String value/)
```

Comment:
rubocop wants a space after the comma and ```%r{}```

Reply:
@ghoneycutt Are you on this, else, I can try to find some time!?